ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -67,8 +67,8 @@ tests:
 - as: installer-rehearse-vsphere
   interval: 960h
   steps:
-    cluster_profile: vsphere
-    workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
+    cluster_profile: vsphere-discon
```

Comment:
How about adding a dedicated test for vsphere-discon cluster_profile in the 4.12.yaml just like what we did in 4.10

Reply:
But it is okay for this PR. We can file another PR to follow up that.

Reply:
Good suggestion! We can add that in next PR, just keep this one for disconnected profile and add another one for non-disconnected.